website/content/docs/templates/hcl_templates/contextual-variables.mdx

```diff
@@ -129,3 +129,24 @@ null.first-example: output will be in this color.
 Make sure to wrap your variable in single quotes in order to escape the
 string that is returned; if you are running a dev version of packer the
 parenthesis may through off your shell escaping otherwise.
+
+# HCP Packer Iteration ID
+
+If your build is pushing metadata to the HCP Packer reigstry, this variable is
```

Comment:
What do you think about including an example with the manifest post-processor as well?

```hcl
  post-processor "manifest" {
    output     = "manifest.json"
    strip_path = true
    custom_data = {
      iteration = "${packer.iteration_id}"
    }
  }
```

```shell-session
    {
      "name": "basic-example",
      "builder_type": "file",
      "build_time": 1633962545,
      "files": [
        {
          "name": "compressed_artifact",
          "size": 26
        }
      ],
      "artifact_id": "File",
      "packer_run_uuid": "00821859-e924-1325-13c3-f1e62237875e",
      "custom_data": {
        "iteration": "01FHQW3H5J6F4FYSWQ7DX85N2C"
      }
    }
```

Reply:
will do!